at-dungnguyen/Week3/2018_07_03_at_dungnguyen3_exercise_3.rb

```diff
@@ -0,0 +1,37 @@
+# Write a Ruby data structure to hold this data.
+teachers = [
+  { 
+    name: "Paul Barry", 
+    areas_of_interest: [ "Programming", "Networking", "Security", "Open Source", "Framework" ]
+  },
+  {
+    name: "Chris Meudec", 
+    areas_of_interest: [ "Testing", "Safety Systems", "Formal Systems", "Open Source", "Programming Languages" ]
+  },
+  {
+    name: "Nigel Whytec", 
+    areas_of_interest: [ "Graphics", "Imaging", "Programming", "Sign Language", "Trees" ]

```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/7d13fd74df598a7c917a2c919adafd9a0f832c62/at-dungnguyen/Week3/2018_07_03_at_dungnguyen3_exercise_3.rb#L13"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>